src/PythonClient/tests/quixstreams/test_dataframes/fixtures.py

```diff
@@ -106,6 +125,23 @@ def factory(
     return factory
 
 
+@pytest.fixture()
+def producable_row_factory():
```

Comment:
There's already a row_factory in one of the fixture modules. Let's re-use the existing one.

Reply:
This is now cleaned up in latest commit so that everything uses one Row fixture =)